validator/src/cli.rs

```diff
@@ -1537,12 +1537,13 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .help(BlockProductionMethod::cli_message()),
         )
         .arg(
-            Arg::with_name("enable_block_production_forwarding")
-            .long("enable-block-production-forwarding")
+            Arg::with_name("disable_block_production_forwarding")
+            .long("disable-block-production-forwarding")
```

Comment:
+ `.requires("staked_nodes_overrides")`

Reply:
I don't feel strongly we should prevent using the flag here.

If stake overrides is there, we have the flag to prevent forwarding.
If stake overrides is not there, we either have or don't have and forwarding is disabled either way.

Reply:
Okay, seems everyone else feels it should be there, I will add it: [631a88b](https://github.com/anza-xyz/agave/pull/1801/commits/631a88bdc428ec6ba1a1fface09e968d9b61e9d8)